Proposals/xgov-6.md

```diff
@@ -0,0 +1,108 @@
+---
+period: 8
+title: Transaction Builder UI
+author: No-Cash-7970 (@No-Cash-7970)
+company_name: <None>
+category: Tools
+focus_area: User Onboarding
+open_source: Yes
+amount_requested: 1
+status: Draft
```

Comment:
Remember that you need to change this to "Final" if you want your proposal to go through the next voting session.